### ISSUE TITLE
wer is different on test 

### ISSUE THREAD

**Issue by guoying1030** [tensorflow/tensorflow#9006]
When I use the distributed tensorflow and keras combined training, each epoch will be saved on the worker0 model for the keras model, but on the validation set, before saving the model, its wer is 0.17, when saved as keras model, I use a separate program to load keras model, using the same proof set, why its wer becomes 0.22

**Comment by asimshankar** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
